sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/ClassificationDocumentResult.java

```diff
@@ -14,7 +14,7 @@ public class ClassificationDocumentResult extends DocumentResult {
     /*
      * The class property.
      */
-    @JsonProperty(value = "class", required = true)
+    @JsonProperty(value = "classifications", required = true)
```

Comment:
if the service deploys a fix which changes this to start correctly returning `class`, will the java code still work?

Reply:
it won't for sure. Once the service corrects it, will create a PR to address it.